Promise.reject(new Error("THERE IS AN ERROR!!!")).catch(console.log);
